addons/visualization.pictureit/visualization.pictureit.json

```diff
@@ -4,9 +4,8 @@
     "sources": [
         {
             "type": "git",
-            "url": "https://github.com/xbmc/visualization.pictureit",
-            "tag": "20.0.0-Nexus",
-            "commit": "d4361f228ae098ce1ab9ab5acdc5cb0e4c0ea13a"
+            "url": "https://github.com/linuxwhatelse/visualization.pictureit",
```

Comment:
Has been moved to xbmc, I would like to keep it that way. Only a redirect not happening, but still nicer.

Reply:
I can revert that, but it should be updated in the repo as well

Reply:
Yeah, realized that after commenting

Reply:
Upstream has been updated to reflect this